packages/babel-plugin-proposal-class-properties/test/fixtures/regression/7951/output.mjs

```diff
@@ -0,0 +1,9 @@
+export class Foo extends Bar {
```

Comment:
The problem was that when `...args` was renamed to `...args2`, this export declaration was split.
For some reason `classPath.parentPath` was set to `classPath` instead of `programPath`. I'll investigate it, but this PR "workarounds" the problem by only splitting the export declaration when necessary.